monai/handlers/segmentation_saver.py

```diff
@@ -101,13 +101,12 @@ def __init__(
                 output_dir: /output,
                 data_root_dir: /foo/bar,
                 output will be: /output/test1/image/image_seg.nii.gz
-            batch_transform: a callable that is used to transform the
-                ignite.engine.batch into expected format to extract the meta_data dictionary.
-                it can be used to extract the input image meta data: filename, affine, original_shape, etc.
-            output_transform: a callable that is used to transform the
-                ignite.engine.output into the form expected image data.
-                The first dimension of this transform's output will be treated as the
-                batch dimension. Each item in the batch will be saved individually.
+            batch_transform: a callable that is used to extract the `meta_data` dictionary of the input images
```

Comment:
my suggestion is:
batch_transform: a callable that is used to transform the output batch achieved from `ignite.engine.state.batch` to extract the `meta_data` dictionary from the input images. The purpose of extracting the `meta_data` dictionary is to extract some necessary information such as filename, affine, original_shape, etc.

Reply:
updated in the latest commit.
Thanks.